TrippyGL.Fonts.Building/FontBuilderExtensions.cs

```diff
@@ -65,7 +45,7 @@ public static TrippyFontFile CreateFontFile(string fontFile, float size, char fi
         /// </summary>
         public static TrippyFontFile CreateFontFile(Stream fontStream, float size, char firstChar = ' ', char lastChar = '~', Color? backgroundColor = null)
         {
-            return FontBuilder.CreateFontFile(new FontGlyphSource(FontInstance.LoadFont(fontStream), size, firstChar, lastChar), backgroundColor);
+            return FontBuilder.CreateFontFile(new FontGlyphSource(new FontCollection().Add(fontStream).CreateFont(size), firstChar, lastChar), backgroundColor);
```

Comment:
> new FontCollection().Add(fontStream).CreateFont(size)

@JimBobSquarePants Is there a way to create a `Font` instance without having to create a `FontCollection`?

Reply:
Not currently as you need a `FontFamily` which is populated by the collection. 

You could maintain a single collection as a static field.